src/app.scss

```diff
@@ -132,4 +132,7 @@ body {
   & > ol li {
     max-width: none;
   }
-}
+  h1+*, h2+*, h3+*, h4+*, h5+*, h6+* {
```

Comment:
For some reason, this style is stripped when I ran the upgrades.

Reply:
I would just note that in a comment here. Adds additional context if it becomes something to fix/return to in the future